en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -80,142 +76,132 @@ For example, see the video ID circled in red in the illustration below: `q2l-8-r
 
-With YouTube Data Tools, you can use video IDs to pull video metadata and comments. You can also download other files, such as network diagrams of users, videos, and recommended videos. YouTube Data Tools outputs a neatly organized `.csv` spreadsheet of the downloaded comments alongside metadata about the exact time the comment was made, user information, and information about replies. Using this spreadsheet, a simple sort on the **replyCount** column can extract threaded conversations in order to focus on dialogue. The comments alone could also be concatenated into one large text file for topic modeling, vector space modeling, or other corpus analytics. 
+With YouTube Data Tools, you can use video IDs to pull video metadata and comments. You can also download other files, such as network diagrams of users, videos, and recommended videos. YouTube Data Tools outputs a neatly organized CSV spreadsheet of the downloaded comments, alongside metadata about the exact time the comment was made, user information, and information about replies. Using this spreadsheet, a simple sort on the **replyCount** column can extract threaded conversations in order to focus on dialogue. The comments alone could also be concatenated into one large text file for topic modeling, vector space modeling, or other corpus analytics. 
 
 An alternative to using YouTube Data Tools is to obtain YouTube API authorization credentials from YouTube's owner, Google, so that you can query the API directly. This is a multi-step and somewhat complicated process, because the workflow can change each time Google issues updates. If you wish to query the API in R, you can try the [`tuber` package](https://cran.r-project.org/web/packages/tuber/index.html) and register for a Google developer account. While a developer account allows you to incorporate YouTube functionality into your own website or app, it can also be used simply to search and download YouTube content (see YouTube’s API [reference page](https://developers.google.com/youtube/v3/docs) for more information). If you wish to participate in YouTube’s [researcher program](https://research.youtube/how-it-works/), there is a separate application process.
 
-### Downloading Comments and Metadata
+#### Sample Dataset

```

Comment:
I thought it made sense to move this section up before the steps for using YouTube Data Tools, to avoid readers spending unnecessary time reading those sections (irrelevant if downloading the sample dataset).